utilities/logger/src/client.ts

```diff
@@ -0,0 +1,28 @@
+import LoggerInterface from './LoggerInterface';
+
+/**
+ * Client logger implementation
+ */
+
+export class Client implements LoggerInterface {
```

Comment:
do you mean that the implementation has no value or this code has no value? its only a placeholder really implementing interface.